Python/ceval.c

```diff
@@ -44,6 +44,23 @@
 #  error "ceval.c must be build with Py_BUILD_CORE define for best performance"
 #endif
 
+#if !defined(Py_DEBUG)
+// bpo-45116: The MSVC compiler fails to inline these in PGO build,
+// and they're kind of important for performance.
```

Comment:
You might explain that MSVC can inline them, but it's just an arbitrary limit which cannot be configured.

neonene wrote "PR25244 told me the amount of code in _PyEval_EvalFrameDefault() is over the limit of PGO."

I suggest something like:

```suggestion
// bpo-45116: The MSVC compiler does not inline these 3 static inline functions in PGO build
// in _PyEval_EvalFrameDefault(), because this function is over the limit of PGO,
// and these limits which cannot be configured. Define them as macro to make sure that
// there are always inlined by the preprocessor.
```

Maybe tomorrow, MSVC PGO limits will change or become configurable, and it will become possible to remove these macros on more recent MSVC versions.